### PR TITLE
Update Bin Docs

### PR DESCRIPTION
**Pull request by kanitw**
Note that the bin transform is actually still broken, but that would be a separate issue.

### CODE REVIEW

File: site/docs/bin.md [hunk 1]
```diff
@@ -53,26 +31,70 @@ The channel definition's `bin` property is for grouping quantitative, continuous
...
+You can directly bin an `encoding` field by using the `bin` property in a [field definition](encoding.html#field).
 
+{% include table.html props="bin" source="PositionFieldDef" %}
 
-#### Examples
+{:#histogram}
+### Example: Histogram
 
-Given a field with quantitative continuous data values
+<!-- Given a field with quantitative continuous data values
```
Comment (domoritz) [inline site/docs/bin.md]: Why do you remove this?

File: site/docs/bin.md [hunk 2]
```diff
@@ -53,26 +31,70 @@ The channel definition's `bin` property is for grouping quantitative, continuous
...
+{% include table.html props="bin,field,as" source="BinTransform" %}
+
+### Example: Histogram with Bin Transform
+
+Instead of using the `bin` property of a field definition, you can also a bin transform
+to derive a new field (e.g., `bin_IMDB_Rating`), and encode the new field instead.
+
+**TODO: example**
+
+<!--
```
Comment (domoritz) [inline site/docs/bin.md]: I'm okay with having this here as a reminder that this is broken. I will fix the issue soon.

File: site/docs/bin.md (hunk 1 above)
Comment (kanitw) [inline site/docs/bin.md]: I think it's making the page too long, but you can uncomment it.

Comment (domoritz) [inline site/docs/bin.md]: Then delete it ;-)

**Pull request merged**